packages/exhibitor-migration-script/buildinfo.json

```diff
@@ -0,0 +1,6 @@
+{
+    "requires": [
+        "python",
```

Comment:
requires should be used for build time dependencies generally, not runtime dependencies (putting in runtime dependencies makes it so things rebuild more often than they would otherwise)